Tests/MongoSwiftTests/MongoClientTests.swift

```diff
@@ -82,6 +82,22 @@ final class MongoClientTests: MongoSwiftTestCase {
         expect(ids[2]).to(equal(ids[1] + 1))
     }
 
+    func testBound() throws {
+        let elg = MultiThreadedEventLoopGroup(numberOfThreads: 1)
```

Comment:
can we increase the `numberOfThreads` here? Otherwise the event loop group will only have a single event loop in it, and this test would pass every time.